mozilla_django_oidc/auth.py

```diff
@@ -321,8 +323,9 @@ def get_or_create_user(self, access_token, id_token, payload):
             user = self.create_user(user_info)
             return user
         else:
-            LOGGER.debug('Login failed: No user with email %s found, and '
-                         'OIDC_CREATE_USER is False', email)
+            LOGGER.debug('Login failed: No user with %s found, and '
+                         'OIDC_CREATE_USER is False',
+                         self.describe_user_by_claims(user_info))
```

Comment:
I agree that having the email in this method is not useful at all. Probably is something that was left over while `get_or_create_user` was evolving. That said, I don't believe we need another method here. As you mentioned, the email field is only used here so this could be just `claims.get('email', '')`

Reply:
All valid points. I didn't think this thoroughly. Thanks for taking the time explaining this.